tests-functional/clients/services/wakuext.py

```diff
@@ -520,3 +595,70 @@ def send_emoji_reaction_retraction(self, last_emoji_id: str):
         params = [last_emoji_id]
         response = self.rpc_request(method="sendEmojiReactionRetraction", params=params)
         return response
+
+    def emoji_reactions_by_chat_id(self, sender_chat_id: str, limit: int):
+        params = [sender_chat_id, None, limit]
+        response = self.rpc_request(method="emojiReactionsByChatID", params=params)
+        return response
+
+    def emoji_reactions_by_chat_id_message_id(self, sender_chat_id: str, message_id: str):
+        params = [sender_chat_id, message_id]
+        response = self.rpc_request(method="emojiReactionsByChatIDMessageID", params=params)
+        return response
+
+    def get_saved_addresses(self, params=[]):
+        response = self.rpc_request("getSavedAddresses", params)
+        return response
+
+    def get_saved_addresses_per_mode(self, is_test: bool):
+        params = [is_test]
+        response = self.rpc_request("getSavedAddressesPerMode", params)
+        return response
+
+    def upsert_saved_address(self, address_payload: dict):
+        params = [address_payload]
+        response = self.rpc_request("upsertSavedAddress", params)
+        return response
+
+    def delete_saved_address(self, address: str, is_test: bool):
+        params = [address, is_test]
+        response = self.rpc_request("deleteSavedAddress", params)
+        return response
+
+    def remaining_capacity_for_saved_addresses(self, is_test: bool):
+        params = [is_test]
+        response = self.rpc_request("remainingCapacityForSavedAddresses", params)
+        return response
+
+    def set_display_name(self, name: str):
+        params = [name]
+        response = self.rpc_request("setDisplayName", params)
+        return response
+
+    def set_profile_showcase_preferences(self, prefs: dict):
+        response = self.rpc_request("setProfileShowcasePreferences", [prefs])
+        return response
+
+    def get_profile_showcase_preferences(self):
+        params = []
+        response = self.rpc_request("getProfileShowcasePreferences", params)
+        return response
+
+    def create_community_from_payload(self, community: dict):
```

Comment:
Would be nice to define the actual parameters here at some point. Probably no need to to it _right now_, but that ideally we should avoid passing a generic `dict` to services.